easybuild/easyconfigs/e/ESMF/ESMF-7.0.0-foss-2016a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'ESMF'
+version = '7.0.0'
+
+homepage = 'http://sourceforge.net/projects/esmf'
+description = """The Earth System Modeling Framework (ESMF) is software for building and coupling weather,
+ climate, and related models."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s_src.tar.gz' % '_'.join(version.split('.'))]
+
+patches = ['ESMF-6.1.1_libopts.patch']
+
+dependencies = [
+    ('netCDF', '4.4.0'),
+    ('netCDF-Fortran', '4.4.3'),
+    ('netCDF-C++', '4.2'),
+]
+
+prebuildopts = 'ESMF_NETCDF_LIBS="`ncconfig --libs` `nf-config --flibs` `ncxx4-config --libs`" '
+
+buildopts = 'ESMF_NETCDF_INCLUDE=$EBROOTNETCDFMINFORTRAN/include ESMF_NETCDF_LIBS="`ncconfig --libs` `nf-config --flibs` `ncxx4-config --libs`"'
```

Comment:
@SimonPinches should `ESMF_NETCDF_LIBS` be defined both through `prebuildopts` *and* `buildopts`?

one of either could be dropped, no?

also, the `buildopts` line is too long here, so it should be broken up across two lines with `buildopts += '...'` if it stays this long